boards/ek-lm4f120xl/board.c

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (C) 2014 Freie Universität Berlin
```

Comment:
Oh..!! I copied Copyright note from the other existing files. 
I will edit this in accordance to the Coding Convention.

Thanks 
Rakendra